packages/flutter/test/material/stepper_test.dart

```diff
@@ -919,28 +1065,26 @@ testWidgets('Stepper custom indexed controls test', (WidgetTester tester) async
       );
     }
 
-    final ThemeData themeLight = ThemeData.light();
-    final bool material3Light = themeLight.useMaterial3;
+    final ThemeData themeLight = ThemeData(useMaterial3: true);
     await tester.pumpWidget(buildFrame(themeLight));
 
-    final String continueStr = material3Light ? 'Continue' : 'CONTINUE';
-    final String cancelStr = material3Light ? 'Cancel' : 'CANCEL';
+    const String continueStr = 'Continue';
+    const String cancelStr = 'Cancel';
     expect(buttonMaterial(continueStr).color!.value, 0);
-    expect(buttonMaterial(continueStr).textStyle!.color!.value, material3Light ? themeLight.colorScheme.onSurface.withOpacity(0.38).value : 0x61000000);
+    expect(buttonMaterial(continueStr).textStyle!.color!.value, themeLight.colorScheme.onSurface.withOpacity(0.38).value);
```

Comment:
This and similar lines are very long and exceed 80 characters maximum.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters

Reply:
We could get the button's Material in a variable once and check its properties instead of calling `buttonMaterial(continueStr)` on every check and we can also make a `colorScheme` variable instead of calling `themeLight.colorScheme` on every check. 

Something like this:
```dart
expect(_buttonMaterial.color!.value, 0);
expect(_buttonMaterial.textStyle!.color!.value, _colorScheme.onSurface.withOpacity(0.38).value);
```